editor/editor_properties.h

```diff
@@ -487,14 +487,17 @@ class EditorPropertyQuaternion : public EditorProperty {
 	AcceptDialog *warning_dialog = nullptr;
 
 	Label *euler_label = nullptr;
+	VBoxContainer *normalize_quaternion = nullptr;
```

Comment:
```suggestion
	VBoxContainer *normalize_quaternion_btn_container = nullptr;
```
The name should be clearer.

Reply:
However, I felt that the normalization button is overemphasized.

So how about making the letters “Nrm” and “1.0” into icons and placing them to the right of the SpinBox?

Then, it will conflicts with EulerEdit, so I think it would be smartest way is making it function as Nrm button when `is_normalized()` is `false`, and as EulerEdit button when `is_normalized()` is `true` since EulerEdit shouldn't be used for non-normalized quaternions.